logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/Message.kt

```diff
@@ -8,10 +8,10 @@ data class Message(
     val id: String,
     val content: MessageContent,
     val conversationId: ConversationId,
-    val date: String,
-    val senderUserId: UserId,
-    val senderClientId: ClientId,
-    val status: Status
+    val date: String = "",
+    val senderUserId: UserId = UserId("", ""),
+    val senderClientId: ClientId = ClientId(""),
```

Comment:
These will get updated later when the message is sent?

Reply:
AFIK date, senderUserId and senderClientId are not needed when sending the calling message over OTR

Reply:
These are not only OTR messages.

These are messages. Kalium messages. A concept that can be seen as a junction of OTR envelope + decrypted content + `conversation.otr-message-add` event data.

[`clientId` is required for OTR envelopes.](https://github.com/wireapp/generic-message-proto/blob/cbf27536938fee59e90d4a6fb8cb523f2be0d168/proto/otr.proto#L74)

`senderUserId` isn't present in the OTR envelope, but when modeling `Message` and exposing it to consumers of Kalium, we say that we can and will provide the `UserID` of whoever sent the message.

Reply:
This data class is not only used for sending calling messages over OTR. This is used for receiving regular Text messages too.

By making this lines optional, this data class is making it easy to completely ignore the `UserID` and `ClientID` when we sending/receiving any OTR messages.

Reply:
Got it, thanks 👌